options.go

```diff
@@ -105,6 +105,10 @@ func WithRankingScoreVars() GetActivitiesOption {
 	return GetActivitiesOption{makeRequestOption("withScoreVars", true)}
 }
 
+func WithExternalRankingVars(externalRankingVars string) GetActivitiesOption {
```

Comment:
not 100% sure how this is going to be used but since the ranking vars are supposed to be JSON wouldnt it be more ergonomic to have this function accept a `map[string]any` at let it handle the JSON marshalling internally?

Reply:
if i marshall  map[string]any inside the WithExternalRankingVars fn, i'll have to check and return err, but i cannot return err in this fn